### PR TITLE
feat: initial commit

### PR DESCRIPTION
**Pull request by christyjacob4**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,62 @@
...
+
+Appwrite is an open-source backend as a service server that abstract and simplify complex and repetitive development tasks behind a very simple to use REST API. Appwrite aims to help you develop your apps faster and in a more secure way. Use the Flutter SDK to integrate your app with the Appwrite server to easily start interacting with all of Appwrite backend APIs and tools. For full API documentation and tutorials go to https://appwrite.io/docs
+
+![Appwrite](https://appwrite.io/v1/images/console.png)
+
+## Installation
+
+### Gradle
+
+Appwrite's Android SDK is hosted on Maven Central. In order to fetch the Appwrite SDK, add this to your root level `build.gradle(.kts)` file:
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply README.md:18]: Done :+1:

File: src/main/kotlin/io/appwrite/services/Avatars.kt [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+     * /account/sessions endpoint. Use width, height and quality arguments to
+     * change the output settings.
+     *
+     * @param code
+     * @param width
+     * @param height
+     * @param quality
+     * @return [Response]     
+     */
+    @JvmOverloads
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply src/main/kotlin/io/appwrite/services/Avatars.kt:27]: Yes the response object is a generic object. The parsing is left to the consuming code. Like this https://github.com/christyjacob4/appwrite-kotlin-demo/blob/main/src/main/kotlin/StorageTest.kt#L103

File: src/main/kotlin/io/appwrite/Client.kt [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+    }
+
+    /// Your secret JSON Web Token
+    fun setJWT(value: String): Client {
+        config["jWT"] = value
+        addHeader("x-appwrite-jwt", value)
+        return this
+    }
+
+    fun setLocale(value: String): Client {
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply src/main/kotlin/io/appwrite/Client.kt:79]: `header.description` is missing. But other values are present in the spec. :+1:

File: src/main/kotlin/io/appwrite/Client.kt [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+            http = builder.build()
+        } catch (e: Exception) {
+            throw RuntimeException(e)
+        }
+
+        return this
+    }
+
+    fun setEndpoint(endPoint: String): Client {
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply src/main/kotlin/io/appwrite/Client.kt:126]: Done :+1:

File: src/main/kotlin/io/appwrite/Client.kt [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+        return this
+    }
+
+    fun addHeader(key: String, value: String): Client {
+        headers[key] = value
+        return this
+    }
+
+    @Throws(AppwriteException::class)
+    suspend fun call(
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply src/main/kotlin/io/appwrite/Client.kt:137]: Done :+1:

File: src/main/kotlin/io/appwrite/Client.kt [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+
+        return this
+    }
+
+    fun setEndpoint(endPoint: String): Client {
+        this.endPoint = endPoint
+        return this
+    }
+
+    fun addHeader(key: String, value: String): Client {
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply src/main/kotlin/io/appwrite/Client.kt:131]: Done :+1:

File: src/main/kotlin/io/appwrite/services/Account.kt [hunk 7]
```diff
@@ -0,0 +1,431 @@
...
+     * one, by passing an email address and a new password.
+     *
+     * @param email
+     * @param password
+     * @return [Response]     
+     */
+    @JvmOverloads
+    @Throws(AppwriteException::class)
+    suspend fun updateEmail(
+		email: String,
```
Comment (abnegate) [inline src/main/kotlin/io/appwrite/services/Account.kt:74]: Service functions with parameters indentation could be changed to match 4 spaces from function start, parameter list closing line could be changed to align with function start

**Review by abnegate — COMMENTED** [review]
No functional issues that I can see, just a few cosmetics

File: LICENSE.md [hunk 8]
```diff
@@ -0,0 +1 @@
+test test test
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply LICENSE.md:1]: Yea the dev branch was a result of the SDK generator where these values resort to the default ones. 
When I regenerate them from here
https://github.com/appwrite/appwrite/blob/master/app/tasks/sdks.php

File: docs/examples/account/create-recovery.md [hunk 9]
```diff
@@ -0,0 +1,11 @@
+import io.appwrite.Client
+import io.appwrite.services.Account
+
+val client = Client(context)
+  .setEndpoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+  .setProject("5df5acd0d48c2") // Your project ID
+  .setJWT("eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ...") // Your secret JSON Web Token
+
+val accountService = Account(client)
```
[the remark this answers is not in the document]
Comment (christyjacob4) [reply docs/examples/account/create-recovery.md:9]: Done :+1:

File: src/main/kotlin/io/appwrite/services/Account.kt (hunk 7 above)
Comment (christyjacob4) [reply src/main/kotlin/io/appwrite/services/Account.kt:74]: :+1:

**Pull request merged**